### ISSUE TITLE
Solver error in ets:file2tab

### ISSUE THREAD

**Issue by ioolkos** [aggelgian/cuter#120]
`cuter ets file2tab '[filename]' --disable-pmatch` (and also file2tab/2) leads to SOLVER ERRORS for me.

Please tell me if it's NOT useful right now to report "small stuff" like that.

**Comment by kostis** [comment]
Your example is not particularly interesting as far as concolic execution is concerned, because you cannot really expect  that CutEr will be able to come up with a file name that makes ets:file2tab/[1,2] crash.  However, as I wrote in an other issue (#115), CutEr should _not_ be crashing with SOLVER ERRORS, independently of whether the tests make sense or not.  In this respect, we appreciate all reporst of such cases you run across.

This particular SOLVER ERROR was due to the Erlang part of CutEr encoding references and pids and sending these to the SMT solver that does not yet know what to do with them.  A pull request (#121), which is now merged on the master branch, comments out the encoding of references and pids for the time being.  We will uncomment them when we come up with some way to handle these terms in the SMT solver.

Your example now works.  Thanks for your report.  We appreciate it.
